src/st/no0/clock_room.c

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "common.h"
+#include "no0.h"
+
+INCLUDE_ASM("st/no0/nonmatchings/clock_room", func_us_801CCC74);
```

Comment:
This does share some similarities with `EntityClockRoomController` in `bomar` but there is some significant differences still most likely it will need to be its own function